api/cas-server-core-api-events/src/main/java/org/apereo/cas/support/events/dao/CasEvent.java

```diff
@@ -129,22 +139,27 @@ public void putAgent(final String dev) {
         put("agent", dev);
     }
 
+    @JsonIgnore
     public Long getTimestamp() {
         return Long.valueOf(get("timestamp"));
     }
 
+    @JsonIgnore
     public String getAgent() {
         return get("agent");
     }
 
-    public String getId() {
-        return get("id");
+    @JsonIgnore
```

Comment:
Why ignore?

Reply:
These are calculated fields based on the contents of other fields that are going into the JSON. They are not required to persist the object.